Cargo.toml

```diff
@@ -91,9 +91,9 @@ name = "fish_key_reader"
 path = "src/bin/fish_key_reader.rs"
 
 [features]
-default = ["installable"]
+default = ["embed-data"]
 benchmark = []
-installable = ["dep:rust-embed"]
+embed-data = ["dep:rust-embed"]
```

Comment:
> This concern is why I'm not pushing for this to become the default right now - it's still
   > "experimental" so people have time to adjust their plugins.

Yeah it would be nice if we can directly notify the most important plugins
(or better, give them fixes, even if they're untested).
I haven't checked how many are affected.

   > Installable builds never read the vendor directories so nothing changes in that respect.

It seems `~/.local/share/fish/vendor_completions.d` is still in `fish_complete_path` when I run an embed-data build with:

	env -u XDG_DATA_DIRS HOME=$(mktemp -d) fish

so I assume it's not completely ignored?

Reply:
That's $XDG_DATA_HOME.

Reply:
Ah, no, it's $__fish_user_data_dir.

We *really* added too much guff here.

Reply:
Okay, I don't know what to do with that one. I missed it being introduced.

We can either set it to empty too, or keep it (it's in your home dir, so it's more likely that it's up-to-date and usable).